collect_app/src/main/java/org/odk/collect/android/views/CustomSpinner.java

```diff
@@ -0,0 +1,60 @@
+package org.odk.collect.android.views;
+
+import android.content.Context;
+import android.support.v7.widget.AppCompatSpinner;
+import android.util.AttributeSet;
+
+/**
+ * Created by vaibhav on 1/3/18.
+ */
+
+public class CustomSpinner extends AppCompatSpinner {
```

Comment:
What would a more descriptive name be? How about `ScrolledToTopSpinner` or something like that?